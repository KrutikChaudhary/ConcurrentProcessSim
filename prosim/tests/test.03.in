1 2
Proc1 8 1
DOOP 4
LOOP 5
  BLOCK 3
  DOOP 4
END
BLOCK 2
DOOP 3
HALT