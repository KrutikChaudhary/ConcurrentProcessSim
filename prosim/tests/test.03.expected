00000: process 1 new
00000: process 1 ready
00000: process 1 running
00002: process 1 ready
00002: process 1 running
00004: process 1 blocked
00007: process 1 ready
00007: process 1 running
00009: process 1 ready
00009: process 1 running
00011: process 1 blocked
00014: process 1 ready
00014: process 1 running
00016: process 1 ready
00016: process 1 running
00018: process 1 blocked
00021: process 1 ready
00021: process 1 running
00023: process 1 ready
00023: process 1 running
00025: process 1 blocked
00028: process 1 ready
00028: process 1 running
00030: process 1 ready
00030: process 1 running
00032: process 1 blocked
00035: process 1 ready
00035: process 1 running
00037: process 1 ready
00037: process 1 running
00039: process 1 blocked
00041: process 1 ready
00041: process 1 running
00043: process 1 ready
00043: process 1 running
00044: process 1 finished
Process 1: Run time 27, Block time 17, Wait time 0
